### DISCUSSION TITLE
BARAM is such a wonderful software

### DISCUSSION THREAD

**Discussion by OpenFoam-User** [nextfoam/baram#53] [Ideas]
Hello guys

Thank you very much for this software. I've head of it several months ago but never tried it because I never expected it to be a serious project (just look at the README file, there are no screenshots nor videos).

I have installed it today and I was totally impressed by it, it is wonderful and surprisingly is stable so far.

However, in the current state, it lacks the following features:

- It lacks some boundary conditions such as: `empty`, `symmetry`, `cyclicAMI`.
- In transient cases, the only available option for "Pressure-velocity Coupling scheme" are `SIMPLE` and `SIMPLEC`. There is no `PISO` or `PIMPLE`
- There is no way to add a function object.

Besides that, my recommendation is to add screenshot or even better, a small video that showcases the capabilities of BARAM so users can quickly grasp it.

Thank you :)

**Comment by kdg5424** [comment]
Hello @OpenFoam-User. Thank you for having attention on BARAM.

We are providing the features you mentioned.

empty, symmetry, cyclicAMI : We are providing these BC. You can see it with click right button of mouse in the 'Boundary Condition' tab.

![image](https://github.com/nextfoam/baram/assets/133930861/f45ea24b-53f2-4e43-bf87-c294e88c0f26)

Scheme : We are not providing PISO. But If you  select SIMPLE with transient, the scheme will be PIMPLE.
You can see it in the stdout.log file.

![image](https://github.com/nextfoam/baram/assets/133930861/9949865d-1189-4fb7-a206-afb2b92eefec)

Function Object : Honestly, I don't understand what you want for Function Object. But if you mean, you want to monitor something (Like Pressure, Velocity, Temperature of Surfaces, Volumes) you can set it in the 'Monitors' tab.

![image](https://github.com/nextfoam/baram/assets/133930861/5bca98b5-f8e9-47a2-8ef5-a53ed0f4ed18)
![image](https://github.com/nextfoam/baram/assets/133930861/3cfb6d21-f000-4f7e-9048-59e9a5d3ce68)

And we are providing some videos and tutorial guides!!! Please visit the below link

Tutorial Guides : https://baramcfd.org/docs/baramFlow_TutorialGuides/
Youtube : https://www.youtube.com/watch?v=PM8cF9lNoMI&t=30s

Again Thank you for having attention on BARAM.
If you have any questions about BARAM, Please contact us. Don't hesitate it.
